os/linux.cpp

```diff
@@ -8,6 +8,8 @@
 #include <charconv>
 #include <cstdlib>
 #include <cstring>
+#include <optional>
+#include <string_view>
```

Comment:
Are these includes needed?

Reply:
Nope! Removed them! This was originally a lot more complicated than it is now, and I just missed nuking the bonus includes. :P Good catch!